app/controllers/instruments_controller.rb

```diff
@@ -1,4 +1,7 @@
 class InstrumentsController < ApplicationController
+  before_action :authenticate_user!
+  skip_before_action :authenticate_user!, only: [:index, :show]
+
```

Comment:
This should be only for [:new, :create] for now. All visitors should be able to view and list the instruments that are available